src/Typed/Resources/doc/index.rst

```diff
@@ -0,0 +1,148 @@
+Symfony UX Typed
+================
+
+Symfony UX Typed is a Symfony bundle integrating `Typed`_ in
+Symfony applications. It is part of `the Symfony UX initiative`_.
+
+Typed is a complete and easy to use animated typed texts.
+Just enter the strings you want to see typed, and it goes live without complexity.
+
+.. image:: Animation.gif
+  :alt: Typed showing messages
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then install the bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-typed
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ yarn install --force
+    $ yarn encore dev
+
+Also make sure you have at least version 3.0 of
+`@symfony/stimulus-bridge`_ in your ``package.json`` file.
+
+Usage
+-----
+
+Typed works using a list of strings and will manage typing them on your page.
+It comes with a lot of parameters to customize the way the strings are typed: speed, cursor, delays
+and smart backspace are some incredible parameters you can use.
+
+The main usage of Symfony UX Typed is to use its Stimulus controller to initialize Typed:
+
+.. code-block:: twig
+
+    <div>
+        I created this UX component because
+        <span {{ stimulus_controller('symfony/ux-typed/typed', {
+            strings: ['I ❤ Symfony UX', 'Symfony UX is great', 'Symfony UX is easy']
+        }) }}></span>
+    </div>
+
+.. note::
+
+That's it! Typed now shows the messages defined in the list passed as argument.
+You can customize the way those messages are typed.
+Parameters are exactly the same as for the `typed library`_
+
+.. code-block:: twig
+
+    <div>
+        I created this UX component because
+        <span {{ stimulus_controller('symfony/ux-typed/typed', {
+            strings: ['I ❤ Symfony UX', 'Symfony UX is great', 'Symfony UX is easy'],
+            smartBackspace: true,
+            startDelay: 100,
+            backSpeed: 20,
+            backDelay: 100,
+            loop: true,
+            showCursor: true,
+            cursorChar: '✨'
+        }) }}></span>
+    </div>
```

Comment:
I love how you included an animated gif for the readme... and then you show how to exactly re-recreate that in the code. Super easy to understand!

Reply:
Frankly, the Stimulus course on SymfonyCasts inspired me a lot. All the credit goes to you!